src/altinn-studio-cli/Upgrade/Frontend/Fev3Tov4/FooterRewriter/FooterUpgrader.cs

```diff
@@ -26,7 +26,7 @@ public IList<string> GetWarnings()
 
     public void Upgrade()
     {
-        var footerFile = Path.Join(uiFolder, "footer.json");
+        var footerFile = Path.Combine(uiFolder, "footer.json");
```

Comment:
Why?

Reply:
Just for consistency

Reply:
Seems so strenge to switch half of them to the one with surprising behavior.

Reply:
https://learn.microsoft.com/en-us/dotnet/api/system.io.path.combine?view=net-8.0#remarks

> However, if an argument other than the first contains a rooted path, any previous path components are ignored, and the returned string begins with that rooted path component. As an alternative to the Combine method, consider using the Join or TryJoin methods.